fdm-app/app/components/blocks/indicators/field-input-dialog.tsx

```
@@ -0,0 +1,217 @@
+                    </Section>
+
+                    {/* Cultivations using Badge + getCultivationColor */}
+                    <Section label={`Teelten (${cultivations.length})`}>
+                        {cultivations.length === 0 ? (
+                            <p className="text-muted-foreground italic">
+                                Geen teelten geregistreerd
+                            </p>
+                        ) : (
+                            <ul className="flex flex-wrap gap-1.5">
+                                {cultivations.map((c, i) => (
+                                    <li key={i}>
```

Biome (2.4.15)

[error] Avoid using the index of an array as key property in an element.

(lint/suspicious/noArrayIndexKey)